scripts/upgrade

```diff
@@ -54,27 +60,59 @@ sudo touch $final_path/doku.php
 
 # Remove deleted files
 # See https://www.dokuwiki.org/install:unused_files
-grep -Ev '^($|#)' ../sources/data/deleted.files | xargs -I {} sudo rm -vrf $final_path/{}
+if [ -f "../sources/data/deleted.files" ]; then
+	grep -Ev '^($|#)' ../sources/data/deleted.files | xargs -I {} sudo rm -vrf $final_path/{}
+fi
+
+# Change owner for all plugins
+sudo chmod -R 755 $final_path/lib/plugins
 
-# Files owned by root, www-data can just read
-sudo find $final_path -type f -print0 | sudo xargs -0 chmod 0644
-sudo find $final_path -type d -print0 | sudo xargs -0 chmod 0755
-sudo chown -R root: $final_path
+# Update all plugins
+for name_plugin in `sudo -s cat $final_path/lib/plugins/*/plugin.info.txt | grep url | awk -F':' '{print $3}'`;
```

Comment:
Like that?
`for name_plugin in $(sudo -s cat $final_path/lib/plugins/*/plugin.info.txt | grep url | awk -F':' '{print $3}');`